app/jobs/pubsub_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class PubsubJob < ApplicationJob
+  retry_on(StandardError, wait: 5.minute, attempts: 2)
+
+  def perform(*args)
+    # puts("\n [PubsubJob][perform-start] #{args}")
+    sleeping_time = rand(5)
+    puts("\n [PubsubJob][sleeping_time]: #{sleeping_time} => #{(sleeping_time % 2).zero?}")
+    raise(StandardError) if (sleeping_time % 2).zero?
```

Comment:
start = Time.current
end = Time.current
elapsed = end - start

problem: time can move backwards! 

https://blog.dnsimple.com/2018/03/elapsed-time-with-ruby-the-right-way

```rb
t = Process.clock_gettime(Process::CLOCK_MONOTONIC)
# => 2810266.714992
t / (24 * 60 * 60.0) # time / days
# => 32.52623512722222
```